spec/spec_helper.rb

```diff
@@ -2,3 +2,11 @@
   require 'simplecov'
   SimpleCov.start
 end
+
+# Configure VCR cassettes
+# VCR.configure do |config|
+#   config.ignore_hosts 'codeclimate.com' if ENV['CI']
+#   config.cassette_library_dir = File.join(ManageIQ::Providers::Lenovo::Engine.root, 'spec/vcr_cassettes')
+# end
+
+Dir[Rails.root.join("spec/shared/**/*.rb")].each { |f| require f }
```

Comment:
Why spec/shared?  The standard location is spec/support, unless I'm missing something?